lib/asciidoctor/standoc/macros.rb

```diff
@@ -88,6 +88,7 @@ def self.generate_file parent, reader
         fn = save_plantuml parent, reader, localdir
         umlfile = Pathname.new(localdir) + "plantuml" + "#{fn}.pml"
         system "plantuml #{umlfile}"
+        sleep(2) if Gem.win_platform?
```

Comment:
@CAMOBAP795 why do we need this strange line here?

Reply:
This is a workaround for windows issue https://ci.appveyor.com/project/ribose/metanorma-standoc/builds/26662260/job/b6ro2ct4tjh242g3 (ugly workaround but I didn't find anything better for now)

I think under the hood in plantuml something wrong with waiting for process https://github.com/plantuml/plantuml/blob/2c655a6b7481c39a41fc0dc57b561bd1db1ecfe2/src/net/sourceforge/plantuml/cucadiagram/dot/AbstractGraphviz.java#L109

I propose to apply this workaround and fix it later in the scope of another ticket

Reply:
OK, but can you document this status in a code comment on top of this line?

Reply:
Done